cli/cli.js

```diff
@@ -48,7 +48,12 @@ async function build(argv) {
     const TARGET_DIR = path.dirname(argv.target);
     const TARGET_EXT = argv.target.split('.').pop();
     const TARGET_FILE_NAME = path.basename(argv.target, `.${TARGET_EXT}`);
-    const TARGET_TYPE = TARGET_EXT === 'wasm' ? 'STANDALONE' : TARGET_EXT === 'base64' ? 'ENCLAVED' : undefined;
+
+    if(!["wasm", "base64"].includes(TARGET_EXT)){
+        throw new Error(`Unsupported target ${TARGET_EXT}, make sure target ends with .wasm or .base64`);
+    } 
+
+    const TARGET_TYPE = TARGET_EXT === 'wasm' ? 'STANDALONE' : 'ENCLAVED';
```

Comment:
I included these small changes to immediately stop the building if the target is not correct